src/services/refactors/moveToFile.ts

```diff
@@ -1131,14 +1131,22 @@ export function getExistingLocals(sourceFile: SourceFile, statements: readonly S
         const declaration = importFromModuleSpecifier(moduleSpecifier);
         if (
             isImportDeclaration(declaration) && declaration.importClause &&
-            declaration.importClause.namedBindings && isNamedImports(declaration.importClause.namedBindings)
+            declaration.importClause.namedBindings
         ) {
-            for (const e of declaration.importClause.namedBindings.elements) {
-                const symbol = checker.getSymbolAtLocation(e.propertyName || e.name);
+            if (isNamespaceImport(declaration.importClause.namedBindings)) {
+                const symbol = checker.getSymbolAtLocation(declaration.importClause.namedBindings.name);

```

Comment:
I think there’s no need for `checker.getSymbolAtLocation` anywhere in this function. Declarations always have a `.symbol`; you can just do `declaration.importClause.namedBindings.symbol` here, and `.symbol` on each `ImportSpecifier` below. Even the existing variable declaration / binding pattern stuff further down can lose the `getSymbolAtLocation`.